### DISCUSSION TITLE
Is repack support nativewind??

### DISCUSSION THREAD

**Discussion by jaemushim** [callstack/repack#783] [Q&A]
I want css library nativewind(tailwind css)
but nativewind doc guide me only metro config.

Can I use nativewind with repack??

4. Modify your metro config section, can i pass it?
![Screenshot_20241105_075943_Chrome](https://github.com/user-attachments/assets/8bc4b875-ef5c-44ad-88d9-7413bfcba76d)

**Comment by jbroma** [comment]
Hey, looking at NativeWind setup, it most likely won't work out of the box, feel free to open an issue with Feature request for that :)

**Comment by sahajarora1286** [comment]
I see that Repack v5 (in RC as of now) claims to provide support for NativeWind.

**Reply by jbroma** [reply]
hi @sahajarora1286, that's correct, you can follow the instruction from the plugin readme to get NativeWind working with Re.Pack: https://github.com/callstack/repack/tree/main/packages/plugin-nativewind
